headers.yml

```
@@ -0,0 +1,11 @@
+root: true
+name: CERN
+start_year: 2021
+header: |-
+  {comment_start} This file is part of the CERN Indico plugins.
```

```suggestion
  {comment_start} This file is part of the JACoW plugin.
```

(and regenerate the headers)